docs/custom-features/media_privacy.md

```diff
@@ -0,0 +1,34 @@
+# Media privacy
+
+This project uses customised versions of Wagtail's `Image` and `Document` models to store and manage the
+privacy of uploaded media files.
+
+1. The privacy of an image or document cannot be changed manually via the Wagtail UI.
+2. Images and documents are private by default, and will only be served to authenticated users with some level of permission on the media item whilst they have this privacy status (so previewing from Wagtail should still work as expected).
+3. Where the storage backend supports it, files for private images and documents (and any renditions created from the original image) will have their access permissions updated at the storage level, preventing direct file access.
+4. Images and documents only become public when they are referenced by blocks in page content, and that page is published.
+5. When a page is unpublished, any public images or documents referenced solely by that page are made private again (references from other draft pages are ignored, as the media can easily be made public again when the draft page is published).
+
+## Use of Wagtail's reference index
+
+Wagtail's reference index is used to track when images and documents (and other objects) are referenced by pages and other objects.
+
+Population of the reference index is automatically triggered by the `post_save` signal for all registered models.
```

Comment:
how about

```suggestion
The reference index is automatically populated via the `post_save` signal for all registered models.
```

?

Reply:
Applied with slight wording change in [04e31c2](https://github.com/ONSdigital/dis-wagtail/pull/46/commits/04e31c2223dbc6004a1397f19ddaadee04101b93)